Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -4776,5 +4776,42 @@
   {
     "Input": "My card number is 12345678",
     "Results": []
+  },
+  {
+    "Input": "OFFICE-1(516)733-3989",
```

Comment:
Can you add a few more cases to show that your changes handle more general cases?
Ex: "OFFICE - 1(516)733-3989" or "Office:(516)733-3989"

Reply:
ok